feature-contributors/src/main/res/values-zh-rCN/strings.xml

```diff
@@ -0,0 +1,4 @@
+<?xml version="1.0" encoding="utf-8"?>
```

Comment:
Same here.

Reply:
Sorry. I understood what I have to do right now.